prow/sinker/hpa.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: sinker
+  labels:
+    app: sinker
+spec:
+  minReplicas: 1
+  maxReplicas: 4
```

Comment:
I'm not sure sinker is horizontally scalable so we should check that before we start scaling it

Reply:
@JoelSpeed no comment stating otherwise, do you think it might be missing? https://github.com/kubernetes/test-infra/blob/master/prow/cluster/starter.yaml#L232

Reply:
@JoelSpeed removed it either way as I don't know currently.